app/controllers/api/v1/users_controller.rb

```diff
@@ -30,7 +30,7 @@ def create
         return render_error errors: user.errors.to_a if hcaptcha_enabled? && !verify_hcaptcha(response: params[:token])
 
         if user.save
-          session[:user_id] ||= user.id
+          session[:user_id] ||= user.reload.id
```

Comment:
This doesn't seem to work.
Upon creating an account, it does not log you in - it buffers until the page crashes.

Reply:
Hmm it was crashing before I added and the reload fixed it - let me retest

Reply:
Just had to do with a stale session - fixed it by removing the `||=`